translations/cs/LC_MESSAGES/wesnoth-ala.po

```diff
@@ -1812,6 +1827,9 @@ msgid ""
 "things, OH OH OH! We to do so, you to say me who is enemy of Wetspot, and I "
 "to squash him!"
 msgstr ""
+"...chápat... nic nechápat! Ale to být dobře, nezvyknout věcem rozumět, hó hó "
```

Comment:
```suggestion
"... chápat... nic nechápat! Ale to být dobře, nezvyknout věcem rozumět, hó hó "
```

Reply:
so, at least in English, my style is that I prefer not to have any spaces between an ellipsis at the start of a sentence and the word that it leads into; if you want to have a different style guideline for Czech, though, I guess that's okay...

Reply:
Using the the space after an ellipsis we distinguish between a teared sentence (... oblivion) and a cut word (...livion), as I stated in the third paragraph about [our Czech ellipsis style](https://wiki.wesnoth.org/CS:Pravidla#V.C3.BDpustka).